lib/hexpm_web/controllers/package_report_controller.ex

```diff
@@ -10,6 +10,23 @@ defmodule HexpmWeb.PackageReportController do
   @report_bad_version_msg "No release matchs given requirement"
   @report_not_accessible "Requested package report not accessible"
 
+  def new_comment(conn, params) do
+    report_id = params["id"]
+    report = PackageReports.get(report_id)
+    author = conn.assigns.current_user
+
+    comments =
+      %{
+        "report" => report,
+        "author" => author,
+        "text" => params["text"]
+      }
+      |> Comments.new()
+
+    conn
+    |> redirect(to: Routes.package_report_path(HexpmWeb.Endpoint, :show, report.id))
```

Comment:
```suggestion
    redirect(conn, to: Routes.package_report_path(HexpmWeb.Endpoint, :show, report.id))
```

We usually avoid single pipes. Same for `Comments.new` above.